x-pack/plugins/ml/server/routes/job_audit_messages.ts

```diff
@@ -96,4 +97,40 @@ export function jobAuditMessagesRoutes({ router, routeGuard }: RouteInitializati
       }
     )
   );
+
+  /**
+   * @apiGroup Annotations
+   *
+   * @api {put} /api/ml/job_audit_messages/clear_messages/{jobId} Index annotation
+   * @apiName ClearJobAuditMessages
```

Comment:
This needs to be added to `apidoc.json` too.

Reply:
Added in 9998eb65ac3e95f8c8d6d6123330e2f9dd23b45f